docs/src/main/tut/iso.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+title:  "Iso"
+section: "optics"
+scaladoc: "http://julien-truffaut.github.io/Monocle/api/#monocle.PLens"
+source: "https://github.com/julien-truffaut/Monocle/blob/master/example/src/main/scala/monocle/example/LensExample.scala"
+pageSource: "https://raw.githubusercontent.com/julien-truffaut/Monocle/master/docs/src/main/tut/iso.md"
+---
+
+# Iso
+
+An `Iso` is an Optic which converts elements of type `A` into elements of type
+`B` without loss.
+
+Consider these two case classes:
+
+```tut:silent
+case class Person(name: String, firstName : String)
+case class Pers(n: String, f: String)
+```
+
+In order to create an `Iso` between `Person` and `Pers` we need to supply two total functions
+
+* get : Person => Pers
+* reverseGet : Pers => Person
+
+```tut:silent
+import monocle._
+val personPrism = Iso[Person, Pers]((p: Person) => Pers(p.name, p.firstName))((p: Pers) => Person(p.n, p.f))
```

Comment:
In documentation I like to see the types even if they can be inferred as I believe it helps to connect the dots as I cannot get them tooltipped from the screen.